components/graphs/Timeline.tsx

```diff
@@ -47,81 +59,15 @@ const Timeline = (props) => {
     displayRange[1] = maxEndDate;
     displayRange[0] = dayjs();
   }
-  const data = [
-    {
-      x: [displayRange[0]?.format(), displayRange[1]?.format()],
-      y: [1, 1],
-      type: 'scatter',
-      mode: 'lines',
-      line: {
-        width: 24,
-        color: theme.brandDark,
-      },
-    },
-  ];
-  const layout: PlotParams['layout'] = {
-    showlegend: false,
-    //showline: true,
-    //zeroline: true,
-    margin: {
-      l: 0,
-      r: 0,
-      t: 0,
-      b: 12,
-    },
-    xaxis: {
-      range: [plotStartDate?.format(), plotEndDate?.format()],
-      autorange: false,
-      tickformat: '         %Y', // FUUUUUU
-      ticks: '',
-      tickangle: 0,
-      showgrid: false,
-      tickfont: {
-        family: '`${theme.fontFamilyTiny}`',
-        size: 10,
-      },
-    },
-    yaxis: {
-      //ticks: "",
-      visible: false,
-    },
-    plot_bgcolor: theme.themeColors.light,
-    height: 36,
-    autosize: true,
-  };
 
   return (
-    <div>
-      {/* Hide plot for now, the visualisation logic pending */}
-      <div aria-hidden role="presentation" style={{ display: 'none ' }}>
-        <Plot
-          data={data}
-          layout={layout}
-          config={{ staticPlot: true }}
-          style={{ width: '100%' }}
-          useResizeHandler
-        />
-      </div>
-      <StatusTitle>
-        {startDate && `${startDate.format('L')} \u2192 `}
-        {continuous && <Badge>{`${t('action-continuous')}`}</Badge>}
-        {((!startDate && endDate) || (continuous && endDate)) && ' \u2192 '}
-        {endDate && `${endDate.format('L')}`}
-      </StatusTitle>
-    </div>
```

Comment:
@terotik I'm assuming it's alright to remove this, it's been pending for two years 😄

Reply:
Go ahead, if @terotik remembers what the intended use was, maybe you can reference this PR from an Asana task (to see what the code looked like).